examples/gallery/lines/linestrings.py

```diff
@@ -0,0 +1,39 @@
+"""
+Plotting lines with LineString/MultiLineString geometry
+=======================================================
```

Comment:
We should add white spaces here:
```suggestion
Plotting lines with LineString / MultiLineString geometry
=========================================================
```

Without white spaces the title does not fit nicely on the card in the gallery:
![gpd_lines_NO_whitespaces](https://github.com/user-attachments/assets/1fb4f496-b053-454a-ad37-a1ad6e95d52c)